src/main/java/com/limechain/trie/structure/TrieStructure.java

```diff
@@ -400,7 +401,7 @@ public Nibbles nodeFullKeyAtIndex(TrieNodeIndex nodeIndex) {
     }
 
     @NotNull
-    Nibbles nodeFullKeyAtIndexInner(int targetNodeIndex) {
+    public Nibbles nodeFullKeyAtIndexInner(int targetNodeIndex) {
```

Comment:
[Same as above] I can't find the reason this has been made public... maybe forgotten?